src/tests/tablet/schedule/AMeetingCanBeDeleted.java

```diff
@@ -0,0 +1,51 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.common.MeetingMethods;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC13: Verify a user can remove meetings
+ * @author Asael Calizaya
+ *
+ */
+public class AMeetingCanBeDeleted {
+	private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);
+	private List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");
+	private String password = meetingData.get(0).get("Password");
+	private String subject;
+	
+	@BeforeMethod(groups = "ACCEPTANCE")
+	public void creationMeetingPreCondition() {
+		MeetingMethods meeting = new MeetingMethods();
+		subject = meetingData.get(1).get("Subject");
+		String startTime = meetingData.get(1).get("Start time");
+		String endTime = meetingData.get(1).get("End time");
+		String attendee = meetingData.get(1).get("Attendee");
+		String body = meetingData.get(1).get("Body");
+		String organizer = meetingData.get(1).get("Organizer");	
+		meeting.createMeetingFromHome(organizer, subject, startTime, endTime, attendee, body, password);
```

Comment:
The method that I use to create a meeting as a preconditions is on meetingMethods class, I don’t think the steps are necessary in a precondition

From: Anaeli [mailto:notifications@github.com]
Sent: Saturday, March 28, 2015 3:42 PM
To: valesil/RoomManager-Automation
Cc: Asael Calizaya Alcoreza
Subject: Re: [RoomManager-Automation] with corrections on test cases and setInformationMeeting changed to public (#114)


In src/tests/tablet/schedule/AMeetingCanBeDeleted.java<https://github.com/valesil/RoomManager-Automation/pull/114#discussion_r27347771>:

> +public class AMeetingCanBeDeleted {

> +     private ExcelReader excelReader = new ExcelReader(EXCEL_INPUT_DATA);

> +     private List<Map<String, String>> meetingData = excelReader.getMapValues("MeetingData");

> +     private String password = meetingData.get(0).get("Password");

> +     private String subject;

> +

> +     @BeforeMethod(groups = "ACCEPTANCE")

> +     public void creationMeetingPreCondition() {

> +            MeetingMethods meeting = new MeetingMethods();

> +            subject = meetingData.get(1).get("Subject");

> +            String startTime = meetingData.get(1).get("Start time");

> +            String endTime = meetingData.get(1).get("End time");

> +            String attendee = meetingData.get(1).get("Attendee");

> +            String body = meetingData.get(1).get("Body");

> +            String organizer = meetingData.get(1).get("Organizer");

> +            meeting.createMeetingFromHome(organizer, subject, startTime, endTime, attendee, body, password);

instead of do this ...it should be
HomeTabletPage homePage = new HomeTabletPage();
SchedulePage schedulePage = homePage .clickScheduleBtn()
schedulePage.createMeeting....

—
Reply to this email directly or view it on GitHub<https://github.com/valesil/RoomManager-Automation/pull/114/files#r27347771>.